docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
+services:
+  ntt:
+    depends_on:
+        - postgres
+    build:
+      context: .
+    restart: on-failure
+    environment:
+      - NEW_BUILD=true
+      - POSTGRES_HOST=postgres
```

Comment:
Does it make a difference if `POSTGRES_HOST` is an environment variable on the `ntt` and not in the `postgres` container?

Reply:
it doesn't as the postgres_host is for the ntt container and not the Postgres container.